unpick-definitions/set_block_state_flags.unpick

```diff
@@ -1,5 +1,6 @@
 v2
 
+#flag set_block_state_flags net/fabricmc/yarn/constants/SetBlockStateFlags DEFAULT
```

Comment:
Unpick doesn't seem to handle this very well yet, so I commented this out for now.

Reply:
Humm, yeah.. Might need to take a look at unpick to handle this.